rascal-vscode-extension/src/RascalExtension.ts

```diff
@@ -146,13 +146,11 @@ export class RascalExtension implements vscode.Disposable {
                 }
                 progress.report({increment: 20, message: "Requesting IDE configuration"});
                 const serverConfig = await rascal.sendRequest<IDEServicesConfiguration>("rascal/supplyIDEServicesConfiguration");
-                progress.report({increment: 25, message: "Calculating project class path"});
-                const compilationPath = await rascal.sendRequest<string[]>("rascal/supplyProjectCompilationClasspath", { uri: uri?.toString() });
```

Comment:
Are there other parts where we use this? if not, let's remove our extension of the lsp protocol.